java-checks/src/main/java/org/sonar/java/checks/EnumMutableFieldCheck.java

```diff
@@ -65,6 +64,25 @@ public void visitNode(Tree tree) {
       }
     }
   }
+  
+  private static boolean isNotStaticOrFinal(VariableTree variableTree, ModifiersTree modifiersTree) {
```

Comment:
Indeed, your commit fixing the sonar issue is apparently not part of the PR. No idea why,